templates/ts-fastify-mongodb-backend/README.md

```diff
@@ -0,0 +1,85 @@
+## Graphback MongoDB + Apollo Server Template
+
+A template that provides you with an easy setup for your application's backend using MongoDB and Apollo server with TypeScript.
+
+### Quickstart Guide
+
+The project has been created using Graphback. Run the project using the following steps:
+
+- Start the database
+
+```
+docker-compose up -d
+```
+
+- Inspect your schema in the `model/datamodel.graphql` file.
+- Start the server
+
+```
+npm run develop
+```
+
+Or, if using yarn
+
+```
+yarn develop
+```
+
+For more on customizing your Graphback application, check out [our docs](https://graphback.dev/docs/gettingstarted)
+
+If your project contains the client application then please follow [`./client/README.md`](./client/README.md) for info regarding running the client-side.
+
+### Dependencies and Tools
+
+The following tools and technologies have been used to create this template:
+
+- [GraphQL](https://graphql.org/): GraphQL is an open-source data query and manipulation language for APIs which was publically released by Facebook in 2015.
+
+- [Apollo Server](https://www.apollographql.com/docs/apollo-server/): Apollo Server provides a way to simplify building the GraphQL server. It can be used with several popular libraries for Node.js like Express, Koa, Hapi. Here we have used it with Express.
+
+- [Express](https://expressjs.com/): Express is a minimal and flexible Node.js web application framework that makes building a Node.js server easier by providing a wide range of features.
```

Comment:
We are not using Apollo and Express, this section should talk about Fastify and Mecurus instead.